libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/KotlinPlugin.kt

```diff
@@ -948,7 +951,7 @@ abstract class AbstractAndroidProjectHandler(private val kotlinConfigurationTool
 
     private val BaseVariant.javaTaskProvider: TaskProvider<out AbstractCompile>
         get() = getJavaTaskProvider()
-            // for older versions, get the task instance and wrap it into a TaskProvider
+        // for older versions, get the task instance and wrap it into a TaskProvider
             ?: getJavaTask(this).let { it.project.tasks.withType(AbstractCompile::class.java).named(it.name) }
```

Comment:
if we use gradle 5.1 apis, this can be shortened to `named(Class, String)`. Or we need to keep using this "old" api because this case is a backward compatibility case ?